src/title/TitlePrivate.js

```diff
@@ -43,7 +43,7 @@ export default function TitlePrivate(props: Props) {
       <View style={componentStyles.inner}>
         <UserAvatarWithPresenceById size={32} userId={user.user_id} />
         <ViewPlaceholder width={8} />
-        <View>
+        <View style={{ flex: 1 }}>
```

Comment:
```
tweak: Handle the overlap part of user's name with InfoButton

Tell the View to take as much space as it's parent has available.
Now the user's name does not overlap with InfoButton.
Truncated part looks well on both landscape and portrait mode.

Fixes: #4469
```

In the summary line, let's say `TitlePrivate` instead of `tweak`; I think that aligns better with the role of the prefix as it's described in [this doc about commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages):

> [The summary] is clear about what part of the code is affected – often by prefixing with the name of the subsystem and a colon, like “zjsunit: …” or “docs: …”

I guess `TitlePrivate` isn't quite as broad as an entire subsystem, but I'm not sure what subsystem would be more helpful to mention, and I'm in the habit of just using component files' names in small commits, like this one, that just touch one file. 🙂

Let's put a period at the end of the summary line.

In the body, I think it'd be helpful to include a few important facts about what the new style does:

- It makes the `View` take up as much _horizontal_ space as is available. (It's horizontal, instead of vertical, because of the `flexDirection: 'row'` in the `View`'s parent `View`. If you want, you can try removing that `flexDirection: 'row'` and see what happens; the value for that attribute [defaults to 'column'](https://reactnative.dev/docs/flexbox). 🙂)
- It makes the `View` take up as much horizontal space as is available _and no more_. The upper limit is what stops the overlap from happening.

Reply:
Ah, also—since there was some discussion on CZO that led to this solution, let's include a link to that in the commit message.